docs/MANPAGE_SYNTAX.md

```diff
@@ -29,45 +29,51 @@ podman\-command - short description
+It should also be specified if the command can only be run as root. In addition, it should be described when a command, OPTION, or other content cannot be executed with the remote client or in combination with other commands, OPTIONS, or contents. In this case the following sentence is put at the end of the first chapter of an command, OPTION, or content: *IMPORTANT: This option/command/other is not available with the command/OPTION/content/remote Podman client*. For a command, this should be done in the DESCRIPTION part. For the OPTIONS, it should be done in the DESCRIPTION of the specified OPTION. Do not use pronouns in the man pages, especially the word `you`.
 
 ## OPTIONS
-All flags are referred to as OPTIONS. The term flags should not be used. All OPTIONS are listed in this section. OPTIONS that appear in descriptions of other OPTIONS and sections retain their appearance, for example: **--exit**. Each OPTION should be explained to the fullest extend below the OPTION itself. Each OPTION is behind an H4-header (`####`).
+All flags are referred to as OPTIONS. The term flags should not be used. All OPTIONS are listed in this section. OPTIONS that appear in descriptions of other OPTIONS and sections retain their appearance, for example: **--exit**. OPTIONS that are quoted from other manpages have to be linked to those with the command to execute them. OPTIONS within the same manpage don't have to be linked. This ensures clarity. Each OPTION should be explained to the fullest extend below the OPTION itself. Each OPTION is behind an H4-header (`####`). If the OPTION has a default argument, is has to be written in the description of the OPTION. If the OPTION is also not available with the remote client, the sentence about the default argument should the second to last sentence.
 
-#### **--option**, **-o**
 
-OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front.
+#### **--version**, **-v** [=**false** | *true*]
 
-#### **--exit**
+OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front. If the OPTION is boolean, it is not requiered to set an argument. In this case the equal sign is put in brackets `[]` with the arguments, in difference to arguments that are not boolean as shown with **--answer**. The default argument that is shown for a boolean OPTION is given when the OPTION is not used in the command. If the OPTION is used in the command and no argument is specified, the OPTION will always become `true`.\
```

Comment:
There is no reason for `2 (two)`, and it is awkward. Standard usage is to simply use the spelled-out number: "If there are two ..."

Reply:
Typo: `requiered` should be `required`

Reply:
It was an idea. I wanted to see if this a good way to react on reading numbers wrong, but I guess you are right. In the end, it is just useless.